3.6/deployment-dc2-dc-cluster.md

```diff
@@ -11,6 +11,8 @@ _Datacenter to datacenter replication_ requires the `rocksdb` storage engine. Th
 example setup described in this section will have `rocksdb` enabled. If you choose
 to deploy with a different strategy keep in mind to set the storage engine.
 
+ArangoDB is also available as [cloud service - **ArangoDB Oasis**](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic){:target="_blank"}. Fire up your cluster in just a few clicks with the [14-day free trial](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic){:target="_blank"}.
```

Comment:
Ewout says: DC2DC is possible, but not (yet) managed

Reply:
OK. Let's remove it.

Reply:
Removed for now, can be added when DC2DC receives a UI option